common/src/main/java/com/wynntils/services/mapdata/providers/json/JsonAggregatorProvider.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © Wynntils 2025.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.mapdata.providers.json;
+
+import com.wynntils.services.mapdata.features.type.MapFeature;
+import com.wynntils.services.mapdata.providers.builtin.BuiltInProvider;
+import com.wynntils.services.mapdata.type.MapCategory;
+import com.wynntils.services.mapdata.type.MapIcon;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Stream;
+
+/**
+ * A provider that aggregates multiple {@link JsonProvider}s. This is done so that {@link JsonProvider}s
```

Comment:
I'm not sure I get the significance of this. Why do we need this class?

Reply:
Actually, we don't. I just forgot to remove this, once I changed the way I implemented this PR. I'll remove it.